.github/workflows/nightly.yml

```diff
@@ -11,42 +11,101 @@ env:
   SHELL: /bin/bash
 
 jobs:
+  create-draft-release:
+    name: Create Draft GH Release
```

Comment:
These two new jobs should likely be guarded by the same if statement as the `upload-*` jobs:

```
if: github.repository == 'servo/servo' || github.event_name == 'workflow_dispatch'
````